crowbar_framework/config/routes.rb

```diff
@@ -24,104 +24,105 @@
-  post "utils/repositories/activate_all", controller: "repositories", action: "activate_all", as: "activate_all_repositories"
-  post "utils/repositories/deactivate_all", controller: "repositories", action: "deactivate_all", as: "deactivate_all_repositories"
+  get "utils/files/:id(.:format)", controller: "support", action: "destroy", constraints: { id: /[^\/]+/ }, as: "utils_files"
+  get "utils/chef(.:format)", controller: "support", action: "export_chef", as: "export_chef"
+  get "utils/supportconfig(.:format)", controller: "support", action: "export_supportconfig", as: "export_supportconfig"
+  get "utils/:controller/1.0/export(.:format)", action: "export", as: "utils_export"
+  get "utils/:controller/1.0(.:format)", action: "utils", as: "utils_barclamp"
+  get "utils/import/:id(.:format)", controller: "support", action: "import", constraints: { id: /[^\/]+/ }, as: "utils_import"
+  get "utils/upload/:id(.:format)", controller: "support", action: "upload", constraints: { id: /[^\/]+/ }, as: "utils_upload"
+  get "utils/repositories(.:format)", controller: "repositories", action: "index", as: "repositories"
+  post "utils/repositories/sync(.:format)", controller: "repositories", action: "sync", as: "sync_repositories"
+  post "utils/repositories/activate(.:format)", controller: "repositories", action: "activate", as: "activate_repository"
+  post "utils/repositories/deactivate(.:format)", controller: "repositories", action: "deactivate", as: "deactivate_repository"
+  post "utils/repositories/activate_all(.:format)", controller: "repositories", action: "activate_all", as: "activate_all_repositories"
+  post "utils/repositories/deactivate_all(.:format)", controller: "repositories", action: "deactivate_all", as: "deactivate_all_repositories"
 
   # barclamps
-  get "crowbar/:controller/1.0/help", action: "help", as: "help_barclamp"
-  get "crowbar/:controller/1.0/proposals/nodes", action: "nodes", as: "barclamp_nodes"
-  put "crowbar/:controller/1.0/proposals", action: "proposal_create", as: "create_proposal_barclamp"
-  get "crowbar/:controller/1.0/proposals", action: "proposals", as: "proposals_barclamp"
-  post "crowbar/:controller/1.0/proposals/commit/:id", action: "proposal_commit", as: "commit_proposal_barclamp"
+  get "crowbar/:controller/1.0/help(.:format)", action: "help", as: "help_barclamp"
+  get "crowbar/:controller/1.0/proposals/nodes(.:format)", action: "nodes", as: "barclamp_nodes"
+  put "crowbar/:controller/1.0/proposals(.:format)", action: "proposal_create", as: "create_proposal_barclamp"
+  get "crowbar/:controller/1.0/proposals(.:format)", action: "proposals", as: "proposals_barclamp"
+  get "crowbar/:controller/1.0/proposals/template(.:format)", action: "proposal_template", as: "template_proposal_barclamp"
+  post "crowbar/:controller/1.0/proposals/commit/:id(.:format)", action: "proposal_commit", as: "commit_proposal_barclamp"
```

Comment:
Metrics/LineLength: Line is too long. [122/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)